src/cdk/testing/test-element.ts

```diff
@@ -113,6 +113,9 @@ export interface TestElement {
   /** Gets the value of a property of an element. */
   getProperty(name: string): Promise<any>;
 
+  /** Sets the value of a property of an input. */
+  setInputValue(value: string): Promise<void>;
```

Comment:
I think this should be an optional method until a future version. It's a breaking change if its required, since people have to update their implementations of `TestElement`